scipy/stats/_stats.pyx

```diff
@@ -513,6 +513,105 @@ cdef double _geninvgauss_pdf(double x, void *user_data) nogil except *:
     return math.exp(_geninvgauss_logpdf_kernel(x, p, b))
 
 
+cdef double _phi(double z) nogil:
+    """estimates the normal PDF. Used in `studentized range`"""
```

Comment:
If this is only an estimate due to finite precision, I would not call it an estimate. "Evaluates"?